build.gradle

```diff
@@ -13,14 +13,18 @@ buildscript {
 
 //add dependency repositories
 repositories {
-	maven {
-		name = "CB Repo"
-		url = "http://chickenbones.net/maven"
-	}
-	maven {
-		name = "JEI Maven"
-		url = "http://dvs1.progwml6.com/files/maven"
-	}
+    maven {
+        name = "CB Repo"
+        url = "http://chickenbones.net/maven"
+    }
+    maven {
+        name = "JEI Maven"
+        url = "http://dvs1.progwml6.com/files/maven"
+    }
+    maven {
+        name = "CurseForge"
+        url = "https://minecraft.curseforge.com/api/maven/"
+    }
```

Comment:
Inventory Tweaks provides an API jar file, so copying `.java` files around isn't needed. Size-wise, the file is only a few kilobytes and adding it via maven coordinates facilitates updating it or including it for different versions. It also looks like the jar method is recommended [here](http://www.minecraftforge.net/forum/topic/54666-solved-1102-how-to-add-an-api/).